### PR TITLE
Adding completed file references to the end of the labs

### PR DESCRIPTION
**Pull request by rochabr**
TL;DR 
Customers are facing many issues with YAML indentation that can halt the session progression if you have too many students with the same problem in different places. 

This fix adds the completed file reference at the end of every lab(apart from lab 6) to allow the student to simply copy its content and replace it in the file.

### CODE REVIEW

**Comment by rochabr** [comment]
@CalinL did you get a chance to look at this?

File: labs/lab01.md [hunk 1]
```diff
@@ -38,6 +38,49 @@ References:
         run: echo 'The time was ${{ steps.hello.outputs.time }}.'   
 ```
 3. Optional remove the `paths` to trigger the workflow on any push to main branch
-4. Commit the changes into the `main` branch 
+4. Commit the changes into the `main` branch
 5. If not step 3), change a file inside the folder [labs](/labs) and commit the changes into the `main` branch
 6. Go to `Actions` and see the details of your running workflow
+
+## Completed files
```
Comment (CalinL) [inline labs/lab01.md:45]:
```suggestion
## 1.3 Final
```

File: labs/lab02.md [hunk 2]
```diff
@@ -73,3 +73,82 @@ on:
 ```
 3. Commit the changes into the `main` branch
 4. Go to `Actions` and see the details of your running workflow
+
+## Completed files
```
Comment (CalinL) [inline labs/lab02.md:77]:
```suggestion
## 2.3 Final
```

File: labs/lab04.md [hunk 3]
```diff
@@ -38,3 +38,110 @@ on:
 > Make sure it is your repository pull request to not propose changes to the upstream repository. From the drop-down list choose the base repository to be yours.
 11. Complete the pull request and delete the source branch
 12. Go to `Actions` and see the details of your running workflow
+
+## Completed files
```
Comment (CalinL) [inline labs/lab04.md:42]:
```suggestion
## 4.2 Final
```

File: labs/lab05.md [hunk 4]
```diff
@@ -96,4 +96,186 @@ on:
         uses: <YOUR-USER-ACCOUNT>/hello-world-javascript-action@v1.1
         with:
             who-to-greet: 'Mona the Octocat'
-```
\ No newline at end of file
+```
+
+## Completed files
```
Comment (CalinL) [inline labs/lab05.md:101]:
```suggestion
## 5.5 Final
```

File: labs/lab07.md [hunk 5]
```diff
@@ -65,3 +65,166 @@ on:
 ```
 4. Commit the changes into the `main` branch
 5. Go to `Actions` and see the details of your running workflow
+
+## Completed files
```
Comment (CalinL) [inline labs/lab07.md:69]:
```suggestion
## 7.3 Final
```

**Review by CalinL — APPROVED** [review]
Approved.

**Pull request merged**